src/goose.rs

```diff
@@ -2832,4 +2967,83 @@ mod tests {
         assert_eq!(goose.request.status_code, 200);
         comment.assert_hits(1);
     }
+
+    #[test]
+    fn test_set_session_data() {
+        #[derive(Debug, PartialEq, Eq, Clone)]
+        struct Foo {
+            bar: String,
+        }
+
+        let foo = Foo {
+            bar: "bar".to_owned(),
+        };
+
+        let configuration = GooseConfiguration::parse_args_default(&EMPTY_ARGS).unwrap();
+        let mut user =
+            GooseUser::single("http://localhost:8080".parse().unwrap(), &configuration).unwrap();
+
+        user.set_session_data(foo.clone());
+
+        let session = user.get_session_data::<Foo>();
+        assert!(session.is_some());
+        assert_eq!(session.unwrap(), &foo);
+
+        let session = user.get_session_data_uncheck::<Foo>();
+        assert_eq!(session, &foo);
+    }
+
+    #[test]
+    fn test_get_mut_session_data() {
+        #[derive(Debug, PartialEq, Eq, Clone)]
```

Comment:
These derives are not needed here, remove.

Reply:
Debug is needed by `assert!` and assert_eq!` macro

Reply:
I removed locally and re-ran tests. It seems it wasn’t necessary?